howto/stream/exchange-oob-data.md

```diff
@@ -314,15 +366,23 @@ amc launch --raw jammy:android12:amd64 \
 Enabling the `allow_custom_system_signatures` feature is required to run the Android application as a system app in an Android container.
 ```
 
-Once the session is active:
+In the [Anbox Streaming SDK](https://github.com/canonical/anbox-streaming-sdk), there is an [out_of_band_v2](https://github.com/canonical/anbox-streaming-sdk/tree/master/examples/android/out_of_band_v2) project. You can either:
```

Comment:
This is what we should start with and then only come to highlights of the application either as part of the steps or separately.